service/src/main/java/bio/terra/workspace/service/crl/CrlService.java

```diff
@@ -128,6 +132,24 @@ public BigQueryCow createBigQueryCow(AuthenticatedUserRequest userRequest) {
     }
   }
 
+  /**
+   * Create a vanilla Bigquery client object, for testing things that aren't in CRL yet.
```

Comment:
Do you want to add another Jira ticket for CRL-ification of this or catch it under the current (PF-906)?